src/containerapp/azext_containerapp/_params.py

```diff
@@ -178,6 +178,9 @@ def load_arguments(self, _):
         c.argument('certificate_file', options_list=['--custom-domain-certificate-file', '--certificate-file'], help='The filepath of the certificate file (.pfx or .pem) for the environment\'s custom domain. To manage certificates for container apps, use `az containerapp env certificate`.')
         c.argument('certificate_password', options_list=['--custom-domain-certificate-password', '--certificate-password'], help='The certificate file password for the environment\'s custom domain.')
 
+    with self.argument_context('containerapp env', arg_group='Peer Authentication') as c:
+        c.argument('mtls_peer_authentication_enabled', arg_type=get_three_state_flag(), options_list=['--mtls-peer-authentication-enabled'], help='Boolean indicating if mTLS peer authentication is enabled for the environment.')
```

Comment:
--enable-xxxx, and consider using a shorter name or add one more short argument

Reply:
this is name suggested by PM.

Reply:
rename to --enable-mtls